### DISCUSSION TITLE
How to edit? If you can't, then why don't add link/option to do so?

### DISCUSSION THREAD

**Discussion by jcubic** [giscus/giscus#182] [Q&A] [answered]
I don't see any information about Editing comments in issues and discussions.

Why don't support the Edit button that will point to the message on GitHub? When one login he needs to grant permission to the GitHub app, so one knows that it's GitHub, maybe just add an option to see the edit button that points to proper GitHub discussions. I've just added a comment on someone's blog and need to search his GitHub to edit a typo, It's not the best UX in my experience. Of course, this should be disabled by default and documented.

PS: I'm learning Next.js and I would be happy to add this feature and contribute to a real Next.js project if you think it's a good idea.

**Comment by laymonage** [comment, marked as answer]
Just like on GitHub, the timestamp is a permalink to the comment on GitHub. You can also click on the number of comments to open the discussion on GitHub.

Yes, I do know that it's not a very obvious feature. There's an issue for this (#36), I think we can start with a "Copy link" menu.
